Docker build option

Now there are instructions for building via installing libraries and via git. I tried both on mac m1 and ubuntu x86 and both crashed. It would be cool if there was a build option in Docker (via docker build ) this would increase the number of people who can fix something and contribute to it

My logs for ubuntu

```bash
git submodule update
Cloning into '/home/alex/Documents/github/pgbouncer/lib'...
Cloning into '/home/alex/Documents/github/pgbouncer/uthash'...
Submodule path 'lib': checked out '46a78d2616ba2e8b5855a3914d5e9375935cf61b'
Submodule path 'uthash': checked out 'eeba1961f203869116a865e57c968e9c86e1b8c4'

alex@alex-GS43VR-7RE:~/Documents/github/pgbouncer$ ./autogen.sh
Using aclocal: 1.16
configure.ac:15: warning: The macro `AC_PROG_CC_STDC' is obsolete.
configure.ac:15: You should run autoupdate.
./lib/autoconf/c.m4:1666: AC_PROG_CC_STDC is expanded from...
lib/m4/usual.m4:69: AC_USUAL_PROGRAM_CHECK is expanded from...
configure.ac:15: the top level

alex@alex-GS43VR-7RE:~/Documents/github/pgbouncer$ ./configure ...
configure: WARNING: you should use --build, --host, --target
checking build system type... Invalid configuration `...': machine `...-unknown' not recognized
configure: error: /bin/bash ./config.sub ... failed

```

> ```shell
> alex@alex-GS43VR-7RE:~/Documents/github/pgbouncer$ ./configure ...
> configure: WARNING: you should use --build, --host, --target
> checking build system type... Invalid configuration `...': machine `...-unknown' not recognized
> configure: error: /bin/bash ./config.sub ... failed
> ```

You seem to have snipped the import parts of the build error

I'm not sure I understood your answer
my full output where i follow readme instruction

```bash
 cd pgbouncer/

alex@alex-GS43VR-7RE:~/Documents/github/pgbouncer$ ls
AUTHORS     config.mak.in  COPYRIGHT             doc  include  Makefile  pyproject.toml  requirements.txt  test            uthash
autogen.sh  configure.ac   dev_requirements.txt  etc  lib      NEWS.md   README.md       src               uncrustify.cfg  win32

alex@alex-GS43VR-7RE:~/Documents/github/pgbouncer$ git submodule init
Submodule 'lib' (https://github.com/libusual/libusual.git) registered for path 'lib'
Submodule 'uthash' (https://github.com/troydhanson/uthash.git) registered for path 'uthash'

alex@alex-GS43VR-7RE:~/Documents/github/pgbouncer$ git submodule update
Cloning into '/home/alex/Documents/github/pgbouncer/lib'...
Cloning into '/home/alex/Documents/github/pgbouncer/uthash'...
Submodule path 'lib': checked out '46a78d2616ba2e8b5855a3914d5e9375935cf61b'
Submodule path 'uthash': checked out 'eeba1961f203869116a865e57c968e9c86e1b8c4'

alex@alex-GS43VR-7RE:~/Documents/github/pgbouncer$ ./autogen.sh
Using aclocal: 1.16
configure.ac:15: warning: The macro `AC_PROG_CC_STDC' is obsolete.
configure.ac:15: You should run autoupdate.
./lib/autoconf/c.m4:1666: AC_PROG_CC_STDC is expanded from...
lib/m4/usual.m4:69: AC_USUAL_PROGRAM_CHECK is expanded from...
configure.ac:15: the top level

alex@alex-GS43VR-7RE:~/Documents/github/pgbouncer$ ./configure ...
configure: WARNING: you should use --build, --host, --target
checking build system type... Invalid configuration `...': machine `...-unknown' not recognized
configure: error: /bin/bash ./config.sub ... failed
```

--- a/README.md
+++ b/README.md
@@ -83,8 +83,8 @@ well as socket activation.  See `etc/pgbouncer.service` and
 Building from Git
 -----------------
 
-Building PgBouncer from Git requires that you fetch the libusual
-submodule and generate the header and configuration files before
+Building PgBouncer from Git requires that you fetch the libusual and
+uthash submodules and generate the header and configuration files before
 you can run `configure`:
 
 	$ git clone https://github.com/pgbouncer/pgbouncer.git
@@ -92,9 +92,14 @@ you can run `configure`:
 	$ git submodule init
 	$ git submodule update
 	$ ./autogen.sh
-	$ ./configure ...
+	$ ./configure
 	$ make
 	$ make install
+
+All files will be installed under `/usr/local` by default. You can
+supply one or more command-line options to `configure`. Run
+`./configure --help` to list the available options and the environment
+variables that customizes the configuration.
 
 Additional packages required: autoconf, automake, libtool, pandoc
 
@@ -113,12 +118,12 @@ Visual $ANYTHING are not supported.
 
 To build on MinGW, do the usual:
 
-	$ ./configure ...
+	$ ./configure
 	$ make
 
 If cross-compiling from Unix:
 
-	$ ./configure --host=i586-mingw32msvc ...
+	$ ./configure --host=i586-mingw32msvc
 
 Running on Windows
 ------------------

